pages/linux/eyewitness.md

```diff
@@ -0,0 +1,24 @@
+# eyewitness
+
+> Python based tool to gather screenshots of websites, as well as additional information such as header info and default credentials if possible.
+> More information: <https://www.christophertruncer.com/eyewitness-triage-tool>.
+
+- Scan a single website using --single flag:
+
+`eyewitness --single {{website}}`
```

Comment:
The command has this case
```suggestion
`EyeWitness --single {{website}}`
```

Reply:
> The command has this case

Yes it is mentioned here https://www.christophertruncer.com/eyewitness-2-0-release-and-user-guide/

Also, `.py` file extension is used

Reply:
Using the version from the Kali repos, it does not require specific capitalization nor the python extension. If we amend the TLDR to reflect this capitalization it will not run on the commandline.

Reply:
in that case we should add an alias page.